### PR TITLE
Invalid permissions setup for the Rust package

### PR DESCRIPTION
**Pull request by dibir-magomedsaygitov**
In scope of this PR we are fixing invalid permissions setup for the Rust package. [Link](https://github.com/actions/virtual-environments/issues/572) to the issue page.

### CODE REVIEW

File: images/linux/scripts/installers/rust.sh [hunk 1]
```diff
@@ -9,8 +9,8 @@ source $HELPER_SCRIPTS/document.sh
 
 set -e
 
-export RUSTUP_HOME=/usr/share/rust/.rustup
-export CARGO_HOME=/usr/share/rust/.cargo
+export RUSTUP_HOME=/etc/skel/rust/.rustup
```
Comment (maxim-lobanov) [inline images/linux/scripts/installers/rust.sh:12]: I think you should also remove logic at the end of this file
image

**Comment by dibir-magomedsaygitov** [comment]
These changes were tested (checked permissions and run rustc) on local machine - everything is working properly.

**Comment by miketimofeev** [comment]
@dibir-magomedsaygitov are you sure that the link provided in the description is correct?

File: images/win/Windows2016-Azure.json [hunk 2]
```diff
@@ -278,6 +278,12 @@
                 "{{ template_dir }}/scripts/Installers/Install-AzureCli.ps1"
             ]
         },
+        {
+            "type": "powershell",
```
Comment (miketimofeev) [inline images/win/Windows2016-Azure.json:282]: why do we need to move Rust installation?

**Comment by dibir-magomedsaygitov** [comment]
> @dibir-magomedsaygitov are you sure that the link provided in the description is correct?

sorry, updated link to the issue

File: images/win/Windows2016-Azure.json (hunk 2 above)
Comment (dibir-magomedsaygitov) [reply images/win/Windows2016-Azure.json]: there is no need in it, i reverted this change

**Pull request closed without merging**